aisynphys/ui/notebook.py

```diff
@@ -506,36 +506,25 @@ def show_distance_profiles(ax, results, colors, class_labels):
     return ax
 
 
-def show_connectivity_profile(x_probed, conn, fit, ax, true_model=None, ymax=None):
+def show_connectivity_profile(x_probed, conn, ax, fit, true_model=None, ymax=None):
     # where to bin connections for measuring connection probability
     x_bins = np.arange(0, 500e-6, 40e-6)
 
     # where to sample models
     #x_vals = 0.5 * (x_bins[1:] + x_bins[:-1])
     x_vals = np.linspace(x_bins[0], x_bins[-1], 200)
-
-    # plot the ground-truth probability distribution (solid green)
-    if true_model is not None:
-        ax.plot(x_vals, true_model.connection_probability(x_vals), color=(0, 0.5, 0))
-
-    # plot the connectivity profile with confidence intervals (black line / grey area)
+   
     _, cprop, lower, upper = connectivity_profile(conn, x_probed, x_bins)
-    ax.plot(x_bins, np.append(cprop, cprop[-1]), drawstyle='steps-post', color=(.5, .5, .5))
-    ax.fill_between(x_bins, np.append(lower, lower[-1]), np.append(upper, upper[-1]), step='post', color=(0.85, 0.85, 0.85))
+    # plot the connectivity profile with confidence intervals (black line / grey area)
+    show_distance_binned_cp(x_bins, cprop, ax, ci_lower=lower, ci_upper=upper)
```

Comment:
Did you make this change, or did I?

Reply:
I did...I broke up the 3 components of `show_connectivity_profile`, the fit, the binned Cp, and the raster into 3 separate functions so that I could use them piecemeal elsewhere. You can still use `show_connectivity_profile` to get exactly what it did before.